legislators-social-media.yaml

```diff
@@ -3478,6 +3480,7 @@
     thomas: '01654'
     govtrack: 400074
   social:
+    twitter: lacyclaymo1
```

Comment:
I'll add that this dataset does frequently fall behind, and we have a fairly sophisticated script to harvest leads on new official accounts, mostly by scraping the HTML of official websites:

https://github.com/unitedstates/congress-legislators/blob/master/scripts/social_media.py

You would be welcome to run that script and use it to identify gaps that we should fill -- there almost certainly are some.